src/store.ts

```diff
@@ -83,8 +69,21 @@ const getFileInternal = (path: string, root: Folder): File | undefined => {
 
   return demand.files[fileName];
 };
+/**
+ * Recursively gets the file for the given path.
+ * Throws FileNotFoundError in case the file metadata not yet provided or the file doesn't exist
```

Comment:
Maybe we can use `@throws` annotation here https://jsdoc.app/tags-throws.html?

Reply:
updated